packages/universe-components/src/table/DonationsTableSortable.tsx

```
@@ -24,43 +25,103 @@
   const donations = props?.donations || [];
 
+  const recs:Donation[] = donations.map(rec => { 
+    const unitValue = rec.impactlinks.length > 0 ? (rec.impactlinks[0].story?.unitvalue||0) : 0
+    let impactScore = ''
+    if(unitValue>0){
+      impactScore = Math.ceil(Number(rec.amount) / unitValue).toString()
+    }
+    const unitLabel = rec.impactlinks.length > 0 ? (rec.impactlinks[0].story?.unitlabel||'') : ''
+    let impactLabel = unitLabel
+    if(unitLabel){
+      impactLabel = unitLabel + (impactScore == '1' ? '' : 's')
```

Biome (1.9.4)

[error] Use === instead of ==. == is only allowed when comparing against `null`

== is only allowed when comparing against null

Using == may be unsafe if you are relying on type coercion
Unsafe fix: Use ===


(lint/suspicious/noDoubleEquals)